doc/api/net.md

```diff
@@ -29,10 +29,10 @@ sockets on other operating systems.
 [`socket.connect()`][] take a `path` parameter to identify IPC endpoints.
 
 On Unix, the local domain is also known as the Unix domain. The path is a
-file system pathname. It gets truncated to an OS-dependent length of
-`sizeof(sockaddr_un.sun_path) - 1`. Typical values are 107 bytes on Linux and
-103 bytes on macOS. If a Node.js API abstraction creates the Unix domain socket,
-it will unlink the Unix domain socket as well. For example,
+file system pathname. It will throw an error when the length of pathname greater
+than the length of `sizeof(sockaddr_un.sun_path)`. Typical values are 107 bytes
+on Linux and 103 bytes on macOS. If a Node.js API abstraction creates the Unix
+domain socket, it will unlink the Unix domain socket as well. For example,
 [`net.createServer()`][] may create a Unix domain socket and
```

Comment:
```suggestion
file system pathname. It will throw an error when the length of pathname is
greater than the length of `sizeof(sockaddr_un.sun_path)`. Typical values are
107 bytes on Linux and 103 bytes on macOS. If a Node.js API abstraction creates
the Unix domain socket, it will unlink the Unix domain socket as well. For
example, [`net.createServer()`][] may create a Unix domain socket and
```
(The `is` in the first line is what I suggest, the other ones are for the character limit on each line.)

Reply:
Done ! Thanks !